core/bin/run

```diff
@@ -2,6 +2,14 @@
 
 #### Housekeeping functions and variables ####################################
 
+if [[ $(date +%u) -eq 3 && $(date +%H) -eq 1 ]]; then
+  echo "I don't run at this time"
+  exit 1
```

Comment:
should our exit code be `0` here? I don't think we consider this an exception, this is our intended functionality during this window.